Makefile

```diff
@@ -25,3 +25,7 @@ pygments:
 
 build:
 	pip install -Ue .[docs,test]
+
+
+clean:
+	git clean -Xf -e "!/*.code-workspace" -e "!/*.vscode"
```

Comment:
Do you still need `-e "!/*.code-workspace" -e "!/*.vscode"` even though they're in gitignore?

The reason I'm asking is because I can see myself removing these after a few months thinking they are redundant. It wouldn't be great if I accidentally break your setup that way.

Reply:
Rephrasing my comment above: those specify *exceptions*. I specifically **don't** want `git clean` to nuke them because cleaning the build does not equate deleting my editor configuration.

Reply:
Apologies, I didn't quite understand how `git clean -X` works.